test/api/tracks_test.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../api_helper'
+
+class TrackssApiTest < Minitest::Test
+  include Rack::Test::Methods
+  include DBCleaner
+
+  def app
+    ExercismAPI::App
+  end
+
+  def setup
+    super
+    @alice = User.create!(username: 'alice', github_id: 1)
+  end
+
+  def test_language_status_returns_404_when_language_does_not_exist
```

Comment:
Is it enough to just test the case in which the track doesn't exist? Why not also test a case where it returns successfully?